flocker/node/agents/ebs.py

```diff
@@ -0,0 +1,229 @@
+# Copyright Hybrid Logic Ltd.  See LICENSE file for details.
+
+"""
+An EBS implementation of the ``IBlockDeviceAPI``.
+"""
+
+from bitmath import Byte, GB
```

Comment:
Thanks for pointing out! Itamar mentioned the same as well a few days ago, i missed following convention during a followup edit. Fixed.